### ISSUE TITLE
Error: EACCES: permission denied, mkdir 'factorio'

### ISSUE THREAD

**Issue by nightcore11** [clusterio/clusterio#653]
Ubuntu server LTS 24.04
Instructions step2 on github:

mkdir clusterio
cd clusterio
npm init "@clusterio"

Unexpected error occurred installing clusterio, please 
report it to https://github.com/clusterio/clusterio/issues 

Error: EACCES: permission denied, mkdir 'factorio'

This does not work with root or non-root users. Manually creating the factorio directory leads to a error saying directory is not empty. I tried this on a fresh VM install of Ubuntu server 24.04 LTS.

**Comment by Hornwitser** [comment]
I'm unable to reproduce this on a fresh VM of Ubuntu 24.04 LTS

```
root# multipass launch 24.04 --name clusterio-test
root# multipass shell clusterio-test
Welcome to Ubuntu 24.04 LTS (GNU/Linux 6.8.0-41-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/pro

 System information as of Sat Aug 31 12:46:07 UTC 2024

  System load:  0.89              Processes:             102
  Usage of /:   43.3% of 3.80GB   Users logged in:       0
  Memory usage: 19%               IPv4 address for ens3: 10.68.9.192
  Swap usage:   0%


Expanded Security Maintenance for Applications is not enabled.

86 updates can be applied immediately.
6 of these updates are standard security updates.
To see these additional updates run: apt list --upgradable

Enable ESM Apps to receive additional future security updates.
See https://ubuntu.com/esm or run: sudo pro status


To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.

ubuntu@clusterio-test:~$ curl -fsSL https://deb.nodesource.com/setup_20.x | sudo -E bash - &&\
sudo apt-get install -y nodejs

[... snipped lots of installation text ...]

ubuntu@clusterio-test:~$ mkdir clusterio
ubuntu@clusterio-test:~$ cd clusterio
ubuntu@clusterio-test:~/clusterio$ npm init "@clusterio"
Need to install the following packages:
@clusterio/create@2.0.0-alpha.18
Ok to proceed? (y) y


> npx
> clusteriocreate

? Operating mode to install Standalone (install both controller and host on this computer)
? Admin account name Hornwitser
? Public DNS/IP address of this server localhost
? (Linux only) Automatically download latest factorio release? Yes
? Plugins to install
info Downloading latest Factorio server release. This may take a while.
info Please wait, installing standalone
info Setting up controller
info Setting up host
info Writing run scripts
Successfully installed standalone
Admin authentication token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJ1c2VyIiwidXNlciI6Ikhvcm53aXRzZXIiLCJpYXQiOjE3MjUxMDg1MzF9.5G9nqkmqdnSd8AQgwGGYgRs5hLgI3s19TbgorIbSjqM
npm notice
npm notice New patch version of npm available! 10.8.2 -> 10.8.3
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.8.3
npm notice To update run: npm install -g npm@10.8.3
npm notice
ubuntu@clusterio-test:~/clusterio$
```

**Comment by nightcore11** [comment]
I suppose I missed the apt-get install -y nodejs. Sorry to jump to the github with a issue. Really appreciate you, thanks for the help.
